x-pack/plugins/task_manager/server/config.ts

```diff
@@ -32,6 +32,12 @@ export const DEFAULT_WORKER_UTILIZATION_RUNNING_AVERAGE_WINDOW = 5;
 export const CLAIM_STRATEGY_UPDATE_BY_QUERY = 'update_by_query';
 export const CLAIM_STRATEGY_MGET = 'mget';
 
+export const DEFAULT_DISCOVERY_INTERVAL_MS = 1000 * 10;
+
+export const DEFAULT_ACTIVE_NODES_LOOK_BACK_S = 30;
```

Comment:
Maybe duration format like `1s` `1h`. we already have `validateDurationSchema`

Reply:
Updated in https://github.com/elastic/kibana/pull/190934/commits/730d9337a47ed52f57fe171e57bbb55fbfb0f920